pages/api/v1/activity-feed/explore-posts.ts

```
@@ -0,0 +1,323 @@
+				proposalHashBlock: proposalHashBlock || null,
+				proposer: proposer || subsquidPost?.preimage?.proposer || otherPostProposer || curator || null,
+				requestedAmount: requestedAmt ? requestedAmt.toString() : undefined,
+				status: status,
+				status_history: statusHistory,
+				summary: '',
+				tally,
+				timeline: proposalTimeline,
+				title: 'Untitled',
+				topic: topicFromType,
+				totalVotes: totalVotes || 0,
+				track_no: !isNaN(trackNumber) ? trackNumber : null,
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)